roles/galaxy-route/defaults/main.yml

```diff
@@ -2,19 +2,20 @@
 # Host to create the root with.
 # If not specific will default to <instance-name>-<namespace>-<routerCanonicalHostname>
 #
-route_host: ''
 
+# common default values
+client_request_timeout: 30
+route_host: ''
 hostname: '{{ deployment_type }}.example.com'
 
-
 # https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size
 nginx_client_max_body_size: 10m
 
-nginx_proxy_read_timeout: 120s
-nginx_proxy_connect_timeout: 120s
-nginx_proxy_send_timeout: 120s
+nginx_proxy_read_timeout: '{{ (([(client_request_timeout | int), 10] | max) / 2) | int }}'
+nginx_proxy_connect_timeout: '{{ (([(client_request_timeout | int), 10] | max) / 2) | int }}'
```

Comment:
currently discussing if the minimum is too large here, debating if waiting 10 seconds for a liveness check seems too long.

Reply:
we're leaving it here now - this could be adjusted in the future though, if we discover this to be an issue.